Storage/SQL/SQLiteHistoryRecommendations.swift

```diff
@@ -18,6 +18,37 @@ extension SQLiteHistory: HistoryRecommendations {
         let thirtyMinutesAgo = NSNumber(unsignedLongLong: now - 30 * microsecondsPerMinute)
         let threeDaysAgo = NSNumber(unsignedLongLong: now - (60 * microsecondsPerMinute) * 24 * 3)
 
+        let blacklistedHosts: [AnyObject?] = [
+            "google.com",
+            "google.ca",
+            "calendar.google.com",
+            "mail.google.com",
+            "mail.yahoo.com",
+            "search.yahoo.com",
+            "localhost",
+            "t.co"
+        ]
+
+        var blacklistSubquery = ""
+        if (blacklistedHosts.count > 0) {
```

Comment:
Nit: `BrowserDB` has a class method named `varlist` that creates the (?,?,?..?) string you need so you don't need to do this here.